stix_shifter_modules/rhacs/stix_transmission/api_client.py

```diff
@@ -13,10 +13,13 @@ def __init__(self, connection, configuration):
         if 'token' in self.auth:
             headers['Authorization'] = "Bearer " + self.auth.get('token')
         url_modifier_function = None
+        #Added self-signed certificate parameter for verification.
         self.client = RestApiClient(connection.get('host'),
                                     connection.get('port', None),
                                     headers,
-                                    url_modifier_function=url_modifier_function
+                                    url_modifier_function=url_modifier_function,
+                                    sni=connection.get('sni'),
+                                    cert_verify=connection.get('selfSignedCert', True)
```

Comment:
This connector use to work with a CA certificate. I think we want this to work with both a CA or a self-signed certificate.

Reply:
Updated the code to support CA and self-signed certificate. Since "sni" and "selfSignedCert" is optional, it is not required to be passed for trusted CA issued server certificate.